tests/test_form_parameters.py

```diff
@@ -1,44 +1,45 @@
-import pytest
```

Comment:
Let me know how to handle this test, it's failing due to the bug I highlighted into the conversation comments.
and I wanna avoid commenting it but I've tried with pytest.mark.xfail but the test raises an error and the marker seems not to work and I don't know how to mark it to be skipped instead of commenting it.

Reply:
@pytest.mark.skip - will skip the test

Reply:
I've tried it as I used to do but the `pytest -n auto --cov ` still raises error for it

Reply:
I will try again, my idea was to use pytest.mark.skipif and check the DRF version, because until 3.11.x, the test will pass

Reply:
Nice idea

Reply:
I've found a solution and I will push it in a while

Reply:
Just pushed the fix and updated docs.
Now pytest won't collect the test if DRF version is greater than 3.11.x. the pytest.mark.skip does not work if there's an error during the collection process.